.github/workflows/php-test.yml

```diff
@@ -35,15 +35,45 @@ on:
       - reopened
       - synchronize
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
 jobs:
   php-test:
-    name: PHP
+    name: PHP ${{ matrix.php }} - WP ${{ matrix.wordpress }}${{ matrix.experimental && ' (experimental)' || '' }}
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    strategy:
+#      fail-fast: true
```

Comment:
```suggestion
```
If we not use `fail-fast` then remove it.

Reply:
This was just leftover from debugging. I need to uncomment it, not remove it. Good catch!